pilot/pkg/networking/core/route/route.go

```diff
@@ -622,8 +615,16 @@ func applyHTTPRouteDestination(
 	}
 
 	var hostnames []host.Name
+	policy := in.Retries
+	if policy == nil {
+		// No VS policy set, use mesh defaults
+		policy = mesh.GetDefaultHttpRetryPolicy()
+	}
+	consistentHash := false
 	if len(in.Route) == 1 {
 		hostnames = append(hostnames, processDestination(in.Route[0], serviceRegistry, listenerPort, hashByDestination, out, action))
+		hash := hashByDestination[in.Route[0]]
+		consistentHash = hash != nil
```

Comment:
How do you handle consitent hash for the below else branch

Reply:
consistent hash weighted desitnations do not work together

Reply:
do we have validation that checked it

Reply:
I do not think we have - but we should add IMO (atleast warn)